src/commands/firestore.js

```diff
@@ -182,9 +182,12 @@ export default function firestoreAction(
     options
   );
 
+  // TODO: refactor to make the initialisation for options and its structure more explicit
```

Comment:
Nice comments! Thanks for adding 💯 

Small change, but it looks like `initialization` is misspelled